composer.json

```diff
@@ -13,14 +13,18 @@
         {
             "name": "Jérémy Marodon",
             "email": "marodon.jeremy@gmail.com"
+        },
+        {
+            "name": "Peter Lee",
+            "email": "peter279k@gmail.com"
         }
     ],
     "require": {
         "php": "^5.6 || ^7.0",
-        "fabpot/goutte": "^3.1"
+        "guzzleHttp/guzzle": "^6.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.4.3",
+        "phpunit/phpunit": "^5.5",
```

Comment:
If in none of the test needs a feature in this [changelog](https://github.com/sebastianbergmann/phpunit/blob/5.5/ChangeLog-5.5.md) please keep 5.4.3, we must remain permissive whenever possible.
The support of PHP5.6 and 7 is bring by 5.0.